tutorials/neurips_challenge_quickstart.md

```diff
@@ -0,0 +1,201 @@
+
+This quick start guide is a short starter guide illustrating the main steps to get started with Lit-GPT, which was selected as the competition's official starter kit. 
+
+
+
+&nbsp;
+
+## Competition Facts
+
+**Permitted GPUs:**
+
+- 1x A100 (40 GB RAM);
+- 1x RTX 4090 (24 GB RAM).
+
+**Permitted models:**
+
+- All transformer-based LLM base models that are not finetuned yet.
+
+The subset of Lit-GPT models supported in this competition is listed in the Table below.
+
+| Models in Lit-GPT         | Reference                                                    |
+| ------------------------- | ------------------------------------------------------------ |
+| Meta AI Llama 2 Base      | [Touvron et al. 2023](https://arxiv.org/abs/2307.09288)      |
+| TII UAE Falcon Base       | [TII 2023](https://falconllm.tii.ae/)     
+| OpenLM Research OpenLLaMA | [Geng & Liu 2023](https://github.com/openlm-research/open_llama) |
+| EleutherAI Pythia         | [Biderman et al. 2023](https://arxiv.org/abs/2304.01373)     |
+| StabilityAI StableLM Base | [Stability AI 2023](https://github.com/Stability-AI/StableLM) |
+
+**Permitted datasets**
+
+Any open-source dataset is allowed. However, [per competition rules](https://llm-efficiency-challenge.github.io/challenge), datasets that utilize "generated content" from other LLMs are not permitted.
+
+Examples of permitted datasets are the following:
+
+- [Databricks-Dolly-15](https://huggingface.co/datasets/databricks/databricks-dolly-15k)
+- [OpenAssistant Conversations Dataset (oasst1)](https://huggingface.co/datasets/OpenAssistant/oasst1)
+- [The Flan Collection](https://github.com/google-research/FLAN/tree/main/flan/v2)
+
+It is allowed to create your own datasets if it is made publicly accessible under an open-source license.
+
+Helpful competition rules relevant to the dataset choice:
+
+- The maximum prompt/completion length the models are expected to handle is 2048 tokens. 
+- The evaluation will be on English texts only.
+
+**Submission deadline**
+
+- October 15, 2023 ([Please check](https://llm-efficiency-challenge.github.io/dates) official website in case of updates.)
+
+&nbsp;
+
+## Lit-GPT Setup
+
+Use the following steps to set up the Lit-GPT repository on your machine.
+
+
+1. Install PyTorch 2.1 (until 2.1 is officially released, you need to install the nightly release):
+
+```bash
+pip install --index-url https://download.pytorch.org/whl/nightly/cu118 --pre 'torch>=2.1.0dev'
+```
+
+2. Clone the repository:
+
+```bash
+git clone https://github.com/Lightning-AI/lit-gpt.git
+```
+
+
+3. Install the dependencies plus some optional ones:
+
+```bash
+cd lit-gpt
+pip install -r requirements.txt tokenizers sentencepiece
+```
+
+
+
+&nbsp;
+
+## Downloading Model Checkpoints
+
+This section explains how to download the StableLM 3B Base model, the smallest model supported in Lit-GPT. (The downloaded and converted checkpoints will occupy approximately 28 Gb of disk space.)
```

Comment:
The smallest model is Pythia right? Curious about the rationale for choosing StableLM

Reply:
Ah you are right, there is a smaller Pythia version. I will rephrase this.

 The rationale

- StableLM is currently the default in most scripts, so the default hyperparameters we have work well for it. 
- I had problems with the Falcon 7B download; StableLM 3B download is more stable for some reason. 

Personally, I think we should replace StableLM as the default but that would be a separate PR. It's also kind of related to #142

Llama 2 is a natural candidate here, but the problem is that this requires a multi-step download procedure, getting permission etc., which is a hassle for a first-time user. We should probably go with OpenLLaMA as the default in Lit-GPT.